### PR TITLE
Remove account indication on password reset

### PR DESCRIPTION
**Pull request by XWB**
Rebase from https://github.com/FriendsOfSymfony/FOSUserBundle/pull/1881

Ping @OskarStark

### CODE REVIEW

File: Controller/ResettingController.php [hunk 1]
```diff
@@ -73,51 +73,42 @@ public function sendEmailAction(Request $request)
             return $event->getResponse();
         }
 
-        if (null === $user) {
-            return $this->render('FOSUserBundle:Resetting:request.html.twig', array(
-                'invalid_username' => $username
-            ));
-        }
+        if (null !== $user && !$user->isPasswordRequestNonExpired($this->container->getParameter('fos_user.resetting.token_ttl'))) {
```
Comment (OskarStark) [inline Controller/ResettingController.php]: looks like this line exceeds 120 chars, or didn't we respect this in this bundle?

File: Resources/translations/FOSUserBundle.en.yml [hunk 2]
```diff
@@ -50,9 +50,12 @@ registration:
 
 resetting:
     password_already_requested: 'The password for this user has already been requested within the last 24 hours.'
-    check_email: 'An email has been sent to %email%. It contains a link you must click to reset your password.'
+    check_email: |
+        An email has been sent. It contains a link you must click to reset your password.
+        Note: You can only request a new password within %tokenLifetime% hours.
+
+        If you don't get an email check your spam folder or try again.
```
Comment (OskarStark) [inline Resources/translations/FOSUserBundle.en.yml]: You can use this for german please!

```
Eine E-Mail wurde verschickt. Sie beinhaltet einen Link zum Zurücksetzen des Passwortes.
Hinweis: Du kannst ein neues Passwort nur alle %tokenLifetime% Stunden beantragen.

Bitte überprüfe deinen Spam-Ordner, falls du die E-Mail nicht erhalten hast.
```

@core23 are you fine with my translation?

Comment (core23) [inline Resources/translations/FOSUserBundle.en.yml]: Not sure if we should use `Du` instead of `Sie`?

Comment (OskarStark) [inline Resources/translations/FOSUserBundle.en.yml]: What about:

```diff
Eine E-Mail wurde verschickt. Sie beinhaltet einen Link zum Zurücksetzen des Passwortes.
- Hinweis: Du kannst ein neues Passwort nur alle %tokenLifetime% Stunden beantragen.
+ Hinweis: Ein neues Passwort kann nur alle %tokenLifetime% Stunden beantragt werden.
-
- Bitte überprüfe deinen Spam-Ordner, falls du die E-Mail nicht erhalten hast.
+ Eventuell wurde diese E-Mail als Spam markiert, wenn sie nicht angekommen ist.
```

Comment (core23) [inline Resources/translations/FOSUserBundle.en.yml]: We should use the last version

File: Controller/ResettingController.php (hunk 1 above)
Comment (XWB) [inline Controller/ResettingController.php]: Fixed.

**Comment by XWB** [comment]
@OskarStark Feel free to review again.

**Pull request merged**